src/toHex.js

```diff
@@ -20,7 +20,18 @@
  * @return {string}
  */
 function toHex(value) {
-  // write code here
+  const hexSymbol = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9',
+    'a', 'b', 'c', 'd', 'e', 'f'];
```

Comment:
лучше так не переносить конечно
```suggestion
  const hexSymbol = [
    '0', '1', '2', '3', '4', '5', '6', '7', '8', '9',
    'a', 'b', 'c', 'd', 'e', 'f'
  ];
```
в идеале - каждый элемент на новой строке, но их очень много